src/pages/Home/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import MovieDetail from '../../components/MovieDetail';
+import MovieList from '../../components/CardList';
+import { getFromStorage } from '../../services';
+
+function Home() {
+  const [movieInfo, setMovieInfo] = useState(getFromStorage('selectedMovie'));
+  const genresLabelList = getFromStorage('genresList');
+  console.log(genresLabelList);
+
+  useEffect(() => {
+  });
+
+  let configParams = {
+    page: '1',
+    isPoster: true,
+  };
+
+  return (
+    <>
+      <MovieDetail movieInfo={movieInfo} />
+      <MovieList id="top_rated" title="Popular on Movy" url="/movie/popular" options={configParams} setMovieInfo={setMovieInfo} />
```

Comment:
the urls should be constants defined in some constant file.

But instead of pass into the list component a url to fetch, let's make it more abstract like the name of the list (ie: popular, terror) and instide the component make the movie fetching based on that.